src/Codeception/Lib/InnerBrowser.php

```diff
@@ -1188,6 +1204,9 @@ public function selectOption($select, $option)
         $formField->select($this->matchOption($field, $option));
     }
 
+    /**
+     * @return mixed|string
```

Comment:
`mixed|string` should be illegal - there is no point in adding `string` to `mixed`.

I looked at the code and, if used properly, this function only returns strings.
$option should be either `string`, `string[]` or associative array containing either `text` or `value` key.

Input validation is non-existent, so it looks like it would return $option as is if it isn't one of above types.  They would likely fail either conversion to string or the following step when DomCrawler tries to select invalid option.